### DISCUSSION TITLE
Costs filtering using Tags

### DISCUSSION THREAD

**Discussion by Azure365Addict** [badlydressedboy/azure-data-costs-ui#4] [Ideas]
Would be great to add costs filtering using Tags. Are there any plans for this?

**Comment by badlydressedboy** [comment]
I am sure I can add that. How would you see it working? Screenshots welcome :-)

**Comment by Azure365Addict** [comment]
My proposal is to add additional "Tag" column in each tab so it would be easier to filter this out?

![image](https://github.com/badlydressedboy/azure-data-costs-ui/assets/78150989/190c37a4-5b0f-474d-9aea-211ba42241e4)

**Comment by badlydressedboy** [comment]
Hi - check the latest release I just uploaded. You can filter by tag and others columns too.

**Reply by Azure365Addict** [reply]
Hi. Thank you very much. Looking great. Makes it a lot easier to get desired data.
